### PR TITLE
podnes-bio

### PR DESCRIPTION
**Pull request by podnesTaF**
### General Checks

- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [ ] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] all links and images work
- [ ] any embedded code snippets are formatted

### CODE REVIEW

**Comment by talmurshidi** [comment]
Hey @podnesTaF and welcome at HYF

You have a nice bio.

I have some notes, please:

1. Assign yourself in `Assignees` section.
2. Add label `student-bio`.
3. Add `Workflows` in `Milestone` section.
4. Create an issue based on [student-bio](https://github.com/HackYourFutureBelgium/workflows/blob/master/deliverables/student-bio-pr.md) and assign it to this PR from `Development` section.

File: student-bios/podnesTaF.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Hi there!
+## I'm Oleksii Pidnebesnyi
+
+<img src="https://th.bing.com/th/id/OIP.p5aepZh8OpbpDE8YlmVtWgHaHa?pid=ImgDet&rs=1" width="100" height="100">
```
Comment (bermarte) [inline student-bios/podnesTaF.md:5]: Hi, check this post here to understand how you should place an image in a markdown file https://stackoverflow.com/questions/41604263/how-do-i-display-local-image-in-markdown

**Review by bermarte — CHANGES_REQUESTED** [review]
You shouldn't use HTML

**Comment by bermarte** [comment]
Like Tamer wrote, you need to complete all the steps relative to the documentation.

**Review by bermarte — CHANGES_REQUESTED** [review]
Once you remove the HTML and make the corrections we asked, we will merge your PR.

File: student-bios/bibhasingh.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Hello :wave:
+# Hello :wave
```
Comment (bermarte) [inline student-bios/bibhasingh.md:1]: You should add only the file with your username. Add one file at the time, please don't use `git add .`

**Review by bermarte — CHANGES_REQUESTED** [review]
you have added an extra file

**Pull request closed without merging**